File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Application/Organizations/Commands/DeleteOrganizationCommand.cs
using Feedback.Analyzer.Domain.Common.Commands;

namespace Feedback.Analyzer.Application.Organizations.Commands;

/// <summary>
/// Represents a command to delete an existing organization.
/// </summary>
public class DeleteOrganizationByIdCommand : ICommand<bool>
{
    /// <summary>
    /// The unique identifier of the organization to be deleted.
    /// </summary>
    public Guid OrganizationId { get; init; }
}

File: src/Insight.Box.FrontEnd/src/modules/organizations/models/DeleteOrganizationCommand.ts
import {Command} from "@/infrastructure/models/command/Command";
import type {Organization} from "@/modules/organizations/models/Organization";

export class DeleteOrganizationCommand extends Command{

    public organizationId:string;

    constructor(organizationId:string) {
        super();
        this.organizationId = organizationId;
    }

}